dashboard/app/models/game.rb

```diff
@@ -224,7 +220,7 @@ def self.setup
         Gamelab:gamelab
         LevelGroup:level_group
         FreeResponse:free_response
-        ScriptCompletion:script_completion
+        NotUsed:not_used
```

Comment:
Can't remove this entry because it would shift the IDs of the `Game` instances that occur after.